src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+const Person = ({ person }) => {
+  let parnterStatus = null;
+
+  if (person.isMarried) {
+    parnterStatus = person.sex === 'm' ? 'wife' : 'husband';
+  }
```

Comment:
```suggestion
    const parnterStatus = person.sex === 'm' ? 'wife' : 'husband';
```

You already have this condition inside JSX so no need to duplicate it